composer-update.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+cd /data
+curl -sS https://getcomposer.org/installer | php
```

Comment:
Rather than re-installing composer in this way every time we want to update our composer dependencies, what would you think of installing composer once, in the Dockerfile?

I think that would be a tweak worth making, since it seems like it would gain us a couple of things:
- We only have to blindly-trust-and-run-whatever's-at-that-URL when building the Docker image, not every time we're updating our dependencies.
- We can still do a composer-update at will, which will get us the latest version more safely (since it can check the updated version's checksum, etc.).

If we do this, we'll also need to tweak the `make clean` command to not blow away the composer.phar file.